### ISSUE TITLE
Can tb-profiler collate include spoligotyping report ?

### ISSUE THREAD

**Issue by ryobon-dev** [jodyphelan/TBProfiler#234]
Hi,

tb-profiler collate is really helpful function in TBprofiler.
However, I noticed that it did not include spoligotyping report.
It would be helpful that you include spoligotyping report in tb-profiler collate.

Can you consider this ?

**Comment by jodyphelan** [comment]
Sure I can add this into the next version.

**Comment by jodyphelan** [comment]
Hi @ryobon-dev 

This has now been added and will be included in the next bioconda release (v4.5.0)

**Comment by ryobon-dev** [comment]
Great !
 
When will you include it in the next bioconda release. 
Can I expect to get it soon ?

Anyway, I eager to use it soon.

**Comment by jodyphelan** [comment]
Yes it should be ready today or tomorrow!

This is what collate looks like at the moment:

```
sample	main_lineage	sub_lineage	spoligotype	DR_type
por5A_illumina_bwa_freebayes_PE	lineage4	lineage4.3.4.2	777777607760400	MDR-TB
por5A_illumina_bwa_gatk_PE	lineage4	lineage4.3.4.2	777777607760400	MDR-TB
por5A_illumina_bwa_bcftools_PE	lineage4	lineage4.3.4.2	777777607760400	MDR-TB
por5A_illumina_bwa_pilon_PE	lineage4	lineage4.3.4.2	777777607760400	MDR-TB
por5A_illumina_bwa_lofreq_PE	lineage4	lineage4.3.4.2	777777607760400	MDR-TB
por5_vcf	lineage4	lineage4.3.4.2		MDR-TB
```

**Comment by jodyphelan** [comment]
Ok it should be available now on bioconda (v4.4.1)
